portfolio/src/main/webapp/style.css

```diff
@@ -51,6 +51,7 @@
   position: fixed;
   top: 0; 
   width: 100%; 
+  z-index: 10; 
```

Comment:
Why is this needed? :) I personally hate z-index:
https://www.reddit.com/r/ProgrammerHumor/comments/58zwnu/the_only_way_to_deal_with_zindex/

Reply:
omg haha

Reply:
that was just so the header would be on top of everything else (and when you scroll the content of the page is below the header) -- but I can look into other ways to do that

Reply:
I tried making the display block (of both the header and the content) and added extra padding to the content. In this case, the header was beneath the content of the page. 
![image](https://user-images.githubusercontent.com/48307028/83259401-9a4ddf00-a17d-11ea-94f3-a73b0426e99d.png)

I also tried using position: relative on the divs but ran into the same issue
![image](https://user-images.githubusercontent.com/48307028/83259535-da14c680-a17d-11ea-913e-6a682d447074.png)

I also tried making the position of the header static (but it scrolls with the page)
(The header is no longer there)
![image](https://user-images.githubusercontent.com/48307028/83259203-3d522900-a17d-11ea-9230-b7c9d60ffcfd.png)

This is what I wanted: (when you scroll, the header stays and the content is hidden beneath)
![image](https://user-images.githubusercontent.com/48307028/83259062-0419b900-a17d-11ea-9974-194e787b1f9e.png)

Do you know any way I can work around using z-index? Thank you!

Reply:
Does "position: sticky" work?

If not, feel free to use "position: fix" and z-index.

Reply:
![image](https://user-images.githubusercontent.com/48307028/83287930-283ebf80-a1a8-11ea-8941-7bd7d8b25f19.png)
ah sticky gives me the same issue where the content as above. 

"position: sticky" with the z-index works though (not sure if this is any better than "position: fixed" though)

Reply:
It is probably not, so either way works for me :)

CSS is hard